.env.sample

```diff
@@ -10,9 +10,17 @@ PIXL_DB_USER=pixl
 PIXL_DB_PASSWORD=
 SKIP_ALEMBIC=false
 
+# Orthanc Raw PostgreSQL instance
+ORTHANC_RAW_DB_HOST=postgres
+ORTHANC_RAW_DB_PORT=5432
+ORTHANC_RAW_DB_NAME=pixl
+ORTHANC_RAW_DB_USER=pixl
+ORTHANC_RAW_DB_PASSWORD=
+
 # Exposed ports
 HASHER_API_PORT=
-POSTGRES_PORT=
+POSTGRES_PIXL_DB_PORT=
```

Comment:
Probably worth moving this elsewhere in the file as this port isn't necessarily what we're exposing it to (we can override this in the system test files, where we are exposing it).

Reply:
Done L13-14: https://github.com/SAFEHR-data/PIXL/blob/b8d5ec35ce2299018f3f83382be51f8f9379d982/.env.sample#L13